bvm/ballerina-core/src/main/java/org/ballerinalang/model/types/BArrayType.java

```diff
@@ -39,6 +39,7 @@
 public class BArrayType extends BType implements BIndexedType {
     private BType elementType;
     private int dimensions = 1;
+    private int size = -1;
```

Comment:
How we handle multi-dimension sealed array? I think we need to store size array here.

Reply:
In a multi-dimensional array, elementType will be an array with a size if that array is also sealed